channels/generic/websockets.py

```diff
@@ -146,9 +146,19 @@ class JsonWebsocketConsumer(WebsocketConsumer):
     error on binary data.
     """
 
+    # Optionally configure a custom json decoder
+    json_decoder = None
+
+    # Optionally configure a custom json encoder
+    json_encoder = None
```

Comment:
Why not have these as functions that just implement `json.loads` and `json.dumps`, then you can drop the if statements below?